ai_services/language/python/languageBatchSamples.py

```diff
@@ -0,0 +1,62 @@
+
+ai_client = oci.ai_language.AIServiceLanguageClient(oci.config.from_file())
+
+compartment_id = "<Provice your compartment Id here>"
+
+language_code = "en"
+
+# Batch Detect Dominant Language
+doc1 = oci.ai_language.models.DominantLanguageDocument(key=key1, text=text1)
+doc2 = oci.ai_language.models.DominantLanguageDocument(key=key2, text=text2)
+documents = [doc1, doc2]
+batch_detect_dominant_language_details = oci.ai_language.models.BatchDetectDominantLanguageDetails(documents=documents,
+                                                                                                   compartment_id=compartment_id)
+output = ai_client.batch_detect_dominant_language(batch_detect_dominant_language_details)
+print(output.data)
+
+doc1 = oci.ai_language.models.TextDocument(key=key1, text=text1, language_code=language_code)
+doc2 = oci.ai_language.models.TextDocument(key=key2, text=text3, language_code=language_code)
+documents = [doc1, doc2]
+
+# Batch Text Classification
+batch_detect_language_text_classification_details = oci.ai_language.models.BatchDetectLanguageTextClassificationDetails(
+    documents=documents, compartment_id=compartment_id)
+output = ai_client.batch_detect_language_text_classification(batch_detect_language_text_classification_details)
+print(output.data)
+
+# Batch Named Entity Recoginiton
+batch_detect_language_entities_details = oci.ai_language.models.BatchDetectLanguageEntitiesDetails(documents=documents,
+                                                                                                   compartment_id=compartment_id)
+output = ai_client.batch_detect_language_entities(batch_detect_language_entities_details)
+print(output.data)
+
+# Batch Key Phrase Detection
+batch_detect_language_key_phrases_details = oci.ai_language.models.BatchDetectLanguageKeyPhrasesDetails(
+    documents=documents, compartment_id=compartment_id)
+output = ai_client.batch_detect_language_key_phrases(batch_detect_language_key_phrases_details)
+print(output.data)
+
+# Aspect based and Sentence level Sentiment Analysis
+batch_detect_language_sentiment_details = oci.ai_language.models.BatchDetectLanguageSentimentsDetails(
+    documents=documents, compartment_id=compartment_id)
+output = ai_client.batch_detect_language_sentiments(batch_detect_language_sentiment_details,
+                                                    level=["ASPECT", "SENTENCE"])
+print(output.data)
+
+# Language Translation
+doc1 = oci.ai_language.models.TextDocument(key=key1, text=text1, language_code=language_code)
+doc2 = oci.ai_language.models.TextDocument(key=key2, text=text2, language_code=language_code)
```

Comment:
can you add a sample for PII as well?
use below text, and add an option to mask with *, leave rest of masking options to default

Hello Support Team, 
I am reaching out to seek help with my credit card number 1234 5678 9873 2345 expiring on 11/23. There was a suspicious transaction on 12-Aug-2022 which I reported by calling from my mobile number (423) 111-9999 also I emailed from my email id sarah.jones1234@hotmail.com. Would you please let me know the refund status?
Regards,
Sarah

Reply:
Done